lua/dylib.lua

```diff
@@ -11,6 +11,12 @@ local ffi = require("ffi")
 -- Structs
 ffi.cdef[[
 
+static const int LIST_IPV4_LPM_TABLE = 1;
```

Comment:
I can't comment at line `module("dylib", package.seeall)` above, but shouldn't we replace `"dylib"` with `...` as in `lua/gatekeeper.lua` since we are adding Lua C functions in the C code before loading `dylib.lua`? I think that if we don't do so, the references to the Lua C functions in the C code won't be available in Lua. In other words, this would be another source for the bug `LUA: dyn_cfg: attempt to call a nil value!`, that is, `list_fib_entries()` itself would be `nil`.

Reply:
When replacing ``"dylib"`` with ``...``, there is an error message ``./lua/dylib.lua:1: bad argument #1 to 'module' (string expected, got nil)!`` when receiving the client message.

Reply:
I added a comment about this issue in a newer review. Comment: https://github.com/AltraMayor/gatekeeper/pull/89#discussion_r236435454

Reply:
I tried to replace ``dylib`` with ``...`` after incorporating the above comment, however, I still get the error message ``./lua/dylib.lua:2: bad argument #1 to 'module' (string expected, got nil)!``

Reply:
Ok. Let's make this pull request work and then look at this issue.

Reply:
Sure.